IVOAArchitecture.tex

```diff
@@ -688,13 +698,13 @@ \subsection{DALI}
 Access Layer (DAL) services. This standard defines the behaviour of common resources, the 
 meaning and use of common parameters, success and error responses, and DAL service 
 registration. The goal of this specification is to define the common elements that are 
-shared across DAL services in order to foster consistency across concrete DAL service 
+shared across DAL seSpectrumrvices in order to foster consistency across concrete DAL service 
```

Comment:
accidental insert here

Reply:
also, I merged my fix of the CI build so if you update vs master the checks should work.